src/components/Table.jsx

```diff
@@ -72,7 +77,30 @@ export default function Table() {
   } = useContext(DashBoardContext);
 
   useEffect(() => {
-    fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=brand`, {
+    const status = [];
+    switch (currentTab) {
+      case "Unassigned":
+        status.push("WAITING_FOR_WRITER");
+        break;
+      case "Completed":
+        status.push("WRITING_COMPLETE", "EDITING_COMPLETE");
+        break;
+      case "Assigned":
+        status.push("ASSIGNED_TO_WRITER", "ASSIGNED_TO_EDITOR");
+        break;
+      case "InProgress":
+        status.push("WRITING_IN_PROGRESS", "EDITING_IN_PROGRESS");
+        break;
+      default:
+        status;
+    }
+    const body = {
+      filters: {
+        status:status
+      }
+    }
+
+    fetch(`${workFlowsUrl}/search?limit=999&page=${currentPage}&unique=brand`, {
```

Comment:
dont hard code 999.
Keep it in  constants file.

Reply:
addressed